extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -545,3 +553,22 @@ export async function promptAndUpdateConnectionStringSetting(projectUri: vscode.
 	}
 	return connectionStringSettingName;
 }
+
+export async function promptConnectionStringPassword(connectionString: string, connectionInfo?: IConnectionInfo): Promise<string> {
```

Comment:
The fact that this is called `promptConnectionStringPassword` but then returns a full connection string is confusing. I'd suggest something more like `fillInConnectionStringPasswordIfNeeded`